.gitignore

```diff
@@ -29,3 +29,6 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+# Ignore Spring files.
+/spring/*.pid
```

Comment:
You've got a bad character at the end of the file See how GH points it out for you. This will be automatically solved by using vim. I know it slows you down in the short term, but it speeds you up in the long term.

Reply:
Bad character?
I figured id learn ruby first, and then get use to vi but i can start using it.